grove-ctrl/README.md

```diff
@@ -0,0 +1,201 @@
+# Demo with a Raspberry Pi and a GrovePi+
+
+This project enables you to use a Raspberry Pi with GrovePi+ as a Thing
+that is managed by Eclipse Ditto and presents information in a WebUI.
+The Thing will provide information about temperature, humidity and illumination.
+Other way round the Thing can accept messages to enable or disable a Buzzer.
+To be able to change the sampling rate, in which sensor values will be retrieved
+and sent to ditto, the Thing is listening to property change events.
+Those property changes can be initiated by the WebUI and will have an effect on
+how often the values are updated. Go on reading, if you want to know more
+about the setup and how to get it running.
+
+![Raspberry Pi with GrovePi+ used gor Grove Control example](docs/grove-ctrl-setup.png)
+
+# Hardware Setup
+
+* GrovePi+
+* Raspberry Pi (tested with Model B+ v1.2)
+* Grove Buzzer (tested with v1.1b)
+* Grove Light Sensor (tested with v1.0)
+* Grove Temperature and Humidity Sensor
+* Machine for running Eclipse Ditto and the WebUI (we used a machine running Linux)
+
+
+# Prerequisites
+
+## Machine
+
+* Up-to-date Web Browser
+* a running Docker daemon (at least version 17.06 CE)
+* Docker Compose installed (at least version 1.14)
+* Apache Maven 3.x installed
+* JDK 8 >= 1.8.0_92 (due to a bug in older versions of the JDK you'll get a compile error)
+
+
+## Raspberry Pi
+
+* Python 3.x installed
+* GrovePi libraries installed (check the manufacturers page on
+how to to this)
+* Raspberry and machine running the same network
+
+# Setup
+
+# Setting up your machine
+
+First, let's start by cloning this project onto the machine:
+
+```bash
+~ $ git clone <project-path>
+~ $ cd ditto-demos/grove-ctrl
```

Comment:
please change to `ditto-examples` - also the other places where there is still "ditto-demos" in this README.md